rule-beans/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: Rule
 Bundle-SymbolicName: supplement.rule.beans;singleton:=true
-Bundle-Version: 8.0.1.qualifier
+Bundle-Version: 8.0.2.qualifier
```

Comment:
this artifact can no longer be the 8 LTS version -> raise to 9.2  ... 
otherwise 8.X users can no longer use the old rule-bean extension.

Reply:
At the moment, everything is still working with 8.0, that's why I just changed it to 8.0.2.
I will have to raise it to 9.2.0 once I change the decision table element to the new extension points.